libs/community/langchain_community/agent_toolkits/openapi/planner_prompt.py

```diff
@@ -84,7 +84,7 @@
 
 Plan: the plan of API calls to execute
 Thought: you should always think about what to do
-Action: the action to take, should be one of the tools [{tool_names}]
+Action: the action to take, 'must be' one of the tools [{tool_names}] without additional words
```

Comment:
I wonder why 'must be' add quotation marks.

Reply:
Tbh i'm not sure and copied the initial fix [here](https://github.com/eunhye1kim/langchain/commit/f7a5ecc077e15e6a33824439f2d925dcb0fcc242) 

I assume this just adds extra emphasis for the LLM to respect this condition -- somewhat similar to saying: "Important: ..."